### DISCUSSION TITLE
CSS is NOT loaded successfullly

### DISCUSSION THREAD

**Discussion by victor10292** [firefly-iii/firefly-iii#9541] [Firefly III]
### Support guidelines

- [x] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [x] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I am using HTTPS to visit firefly. /build/assets/app-DIk1Lb84.css is not loaded. I checked the debug information in Chrome. It seems that the base href is http. How can I fix this issue?

### Debug information

no

**Comment by JC5** [comment]
Hi, thanks for opening a discussion.  

Did you see the documentation?

https://docs.firefly-iii.org/references/faq/docker/#all-links-refer-to-http-so-forms-and-charts-dont-work
